bodhi/tests/server/services/test_updates.py

```diff
@@ -599,6 +601,195 @@ def test_provenpackager_request_privs(self, publish, *args):
         res = app.get('/updates/%s' % str(nvr), status=200)
         eq_(res.json_body['can_edit'], False)
 
+    @mock.patch(**mock_taskotron_results)
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    @mock.patch.dict('bodhi.server.config.config', {'ci.required': True})
+    def test_provenpackager_request_queued_in_ci(self, publish, *args):
+        "Ensure provenpackagers can change the request for any update"
+        nvr = u'bodhi-2.1-1.fc17'
+        user = User(name=u'bob')
+        self.db.add(user)
+        self.db.add(User(name=u'ralph'))  # Add a non proventester
+        self.db.add(User(name=u'someuser'))  # An unrelated user with no privs
+        self.db.flush()
+        group = self.db.query(Group).filter_by(name=u'provenpackager').one()
+        user.groups.append(group)
```

Comment:
This test's name and description say it has something to do with proven packagers, but the user that gets added to the provenpackager group isn't the user that performs actions in the test.